keyboards/tgr/jane/keymaps/default/config.h

```diff
@@ -0,0 +1,19 @@
+/* Copyright 2018 amnesia0287
```

Comment:
what does this file do?

Reply:
The `new_keymap.sh` script copies the default keymap folder if a user runs it. It's basically a placeholder. Same with `keymaps/default/rules.mk`.

Reply:
People will most likely copy and paste these when starting a new keymap if not using the script. Good for placeholders.